TypeCobol/Compiler/Scanner/Scanner.cs

```diff
@@ -2408,13 +2408,13 @@ private bool CheckForPartialCobolWordPattern(int startIndex, out int patternEndI
             // next character must be ':'
             if (line.Length > index && line[index] == ':')
             {
-                patternEndIndex = index;
-                return true;
-            }
-            else
-            {
-                return false;
+                if (compilerOptions.IsCobolLanguage || !(line.Length > index + 1 && line[index + 1] == ':'))
+                {//For Cobol or if ':' is not followed par ':'. Because in TypeCobol we consider :: to be the qualifier symbol.
```

Comment:
typo 'not followed by'

Reply:
@fm-117  It is exactly what I tried to do, I think we should investigate afterwards , what it does not work, it seems that both test will use the same result file, the first result file will win. It is strange for the moment.

Reply:
Type done in 44acf61

Reply:
Ok clearing cache works fine, done in 0386ae2